connectors/automation-anywhere/src/main/java/io/camunda/connector/automationanywhere/model/AutomationAnywhereRequest.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.automationanywhere.model;
+
+import io.camunda.connector.automationanywhere.model.auth.AuthenticationStrategy;
+import io.camunda.connector.http.base.model.HttpCommonRequest;
+import jakarta.validation.Valid;
+
+public class AutomationAnywhereRequest extends HttpCommonRequest {
```

Comment:
Why do you need to inherit here? Cant you just "build" the required http common request in code?